HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Resources/de.lproj/Localizable.strings

```diff
@@ -53,3 +53,9 @@
 "ginihealth.paymentcomponent.installAppBottom.notes.description" = "Hinweis: Bitte aktualisieren oder installieren Sie zunächst die [BANK]-App aus dem App-Store.";
 "ginihealth.paymentcomponent.installAppBottom.tip.description" = "Tipp: Tippen Sie auf 'Weiter', um die Zahlung in der [BANK]-App abzuschließen.";
 "ginihealth.paymentcomponent.installAppBottom.continue.button.text" = "Weiter";
+"ginihealth.paymentcomponent.shareInvoiceBottom.title" = "Rechnungsdaten bereit zum Teilen mit der [BANK]";
```

Comment:
[Suggestion]: @razvancapra could you please add sheet to the keys `ginihealth.paymentcomponent.shareInvoiceBottom` -> `ginihealth.paymentcomponent.shareInvoiceBottomSheet`